modules/post/linux/gather/phpmyadmin_credsteal.rb

```diff
@@ -25,32 +25,49 @@ def initialize(info={})
     ))
    end
 
-  def run
+  def parse_creds(contents)
+    db_user = /\$dbuser=['"](.*)['"];/.match(contents)
```

Comment:
This code block:

```ruby
    db_user = /\$dbuser=['"](.*)['"];/.match(contents)
    db_pass = /\$dbpass=['"](.*)['"];/.match(contents)

    unless db_user && db_pass
      print_error("Couldn't find PhpMyAdmin credentials")
      return
    end

    db_user = db_user[1]
    db_pass = db_pass[1]
```

Could probably be rewritten as:

```ruby
    db_user = contents.scan(/\$dbuser=['"](.*)['"];/).flatten.first
    db_pass = contents.scan(/\$dbpass=['"](.*)['"];/).flatten.first

    unless db_user && db_pass
      print_error("Couldn't find PhpMyAdmin credentials")
      return
    end
```

Also, if the file has been edited manually, it may also make sense to check for `dbpass\s*=\s*` rather than `dbpass=`. Maybe.